tdrs-frontend/src/components/Reports/Reports.test.js

```diff
@@ -385,4 +390,58 @@ describe('Reports', () => {
       getByText('A state, tribe, or territory is required')
     ).toBeInTheDocument()
   })
+
+  it('should show next calander year in fiscal year dropdown in October', () => {
+    const currentYear = new Date().getFullYear()
+
+    const getNow = () => new Date(Date.now())
+
+    jest
+      .spyOn(global.Date, 'now')
+      .mockImplementation(() =>
+        new Date(`October 01, ${currentYear}`).valueOf()
+      )
+    const now = getNow()
+    expect(now).toEqual(new Date(`October 01, ${currentYear}`))
+    const store = mockStore(initialState)
+
+    const { getByLabelText } = render(
+      <Provider store={store}>
+        <Reports />
+      </Provider>
+    )
+
+    const select = getByLabelText('Fiscal Year (October - September)')
+    const options = select.children
+    const expected = options.item(options.length - 1).value
+
+    expect(expected).toEqual((currentYear + 1).toString())
```

Comment:
This checks that the last option in the dropdown is the year after the current year, since that is the new Fiscal year.